src/nitrokey/trussed/admin_app.py

```diff
@@ -150,6 +151,60 @@ def check(cls, i: int, msg: str) -> None:
             raise Exception(f"{msg}: {error}")
 
 
+@enum.unique
+class ConfigFieldType(Enum):
+    BOOL = 0
+    U8 = 1
+
+    @classmethod
+    def from_int(cls, i: int) -> Optional["ConfigFieldType"]:
+        for ty in ConfigFieldType:
+            if ty.value == i:
+                return ty
+        return None
+
+    def is_valid(self, value: str) -> bool:
+        if self == ConfigFieldType.BOOL:
+            return value in ["true", "false"]
+        elif self == ConfigFieldType.U8:
+            try:
+                intval = int(value)
+                if intval < 256 and intval > 255:
```

Comment:
```suggestion
                if intval < 256 and intval >= 0:
```

Or am I missing some Python magic?